earthpy/plot.py

```diff
@@ -431,12 +431,14 @@ def hist(
         A list of title values that should either equal the number of bands
         or be empty. A string is accepted for a single dimension array.
     xlabel : str (optional)
-        The text to print on the x axis
+        The text to print on the x axis.
     ylabel : str (optional)
-        The text to print on the y axis
-    **kwargs : key, value pairings optional
-        These will be passed to the matplotlib .hist call
-        See https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.pyplot.hist.html
+        The text to print on the y axis.
+    range : list (default = [np.nanmin(arr), np.nanmax(arr)])
+        The values over which the histogram will range. Should be
```

Comment:
follow the matplotlib docs but then say we will calculate it they don't provide it using the min and max as you did